framework/src/play/mvc/Scope.java

```diff
@@ -32,6 +32,7 @@
             .equals("true");
     public static boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
             .getProperty("application.session.sendOnlyIfChanged", "false").toLowerCase().equals("true");
+    public static final String SESSION_SAMESITE = Play.configuration.getProperty("application.session.sameSite");
```

Comment:
How about renaming it to "application.session.cookie.sameSite"?

Reply:
> How about renaming it to "application.session.cookie.sameSite"?

Sure, I can do that.

Reply:
Updated PR